modules/gitops-argo-cd-installation.adoc

```diff
@@ -17,10 +17,10 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Select *Openshift GitOps Operator* from the installed operators and select the *Argo CD* tab.
```

Comment:
I realize this isn't in the PR, but wondering if this should be changed to OpenShift rather than Openshift?